src/backend/aspen/covidhub_import/implementation.py

```diff
@@ -120,68 +152,196 @@ def import_project(
+                        (
+                            consensus_genome.recovered_sites
+                            if consensus_genome is not None
+                            else -1,
+                            internal_czb_ids_metadata[czbid][1],
+                            czbid,
+                        )
+                    )
+                    czbids_sorted_by_value_desc.sort(reverse=True)
+                picked_czbid_str = czbids_sorted_by_value_desc[0][2]
+                warnings.warn(
+                    f"External accession {external_accession} represented by multiple"
+                    f" czbids: {czbids}.  Choosing {picked_czbid_str} as the czbid for"
+                    "this sample."
+                )
+                czbids_to_process.append(czbid_lookup_by_str[picked_czbid_str])
+            else:
+                czbids_to_process.append(czbid_lookup_by_str[czbids.pop()])
+
         external_accessions_to_samples: Mapping[str, Sample] = {
             sample.private_identifier: sample
-            for sample in session.query(Sample).filter(
-                and_(
-                    Sample.submitting_group == group,
-                    Sample.private_identifier.in_(external_accessions),
+            for sample in (
+                session.query(Sample)
+                .filter(Sample.submitting_group == group)
+                .options(
+                    joinedload(Sample.uploaded_pathogen_genome)
+                    .joinedload(UploadedPathogenGenome.consuming_workflows)
+                    .joinedload(Workflow.outputs)
                 )
             )
         }
         public_identifier_to_sample: MutableMapping[str, Sample] = dict()
 
         logger.info("Creating new objects...")
-        for consensus_genome in tqdm.tqdm(consensus_genomes):
-            czbid = consensus_genome.sample_fastqs.czb_id
-            if not isinstance(czbid, DphCZBID):
-                warnings.warn(
-                    "consensus genome has non-dphczbid"
-                    f"{consensus_genome.sample_fastqs.czb_id}"
-                )
+        for czbid in tqdm.tqdm(czbids_to_process):
+            if isinstance(czbid, DphCZBID):
+                if czbid.external_accession in (
+                    "OCPHL2207",
+                    "OCPHL2208",
+                ):
+                    external_accession = f"{czbid.external_accession} ({czbid.czb_id})"
+                else:
+                    external_accession = czbid.external_accession
+                collection_date = czbid.collection_date
+                date_received = czbid.date_received or czbid.collection_date
+            elif isinstance(czbid, InternalCZBID):
+                internal_metadata = internal_czb_ids_metadata[czbid.czb_id]
+                external_accession = internal_metadata[0]
+                collection_date = internal_metadata[1]
+                date_received = internal_metadata[1]
```

Comment:
should this be this? 
```suggestion
                date_received = internal_metadata[2]
```

Reply:
no, this is intentionally the same.  we don't really know the collection date, so we're setting it to `date_received`.